Enterprise-CMCS/macpro-onemac
# Oy2 23798 - Subsequent Submissions

Story: https://qmacbis.atlassian.net/browse/OY2-23798
Endpoint: See github-actions bot comment

### Details

Build out new forms for subsequent submissions

### Changes

- All new forms and config; many properties inherited from original form type with overridden properties as needed
- placeholder in one-stream to build parent packages for subsequent submission (no added functionality, only needed now so that submitAny will release lock and return from processing)

### Test Plan

1. Login as state submitter
2. Find any package type (SPA or Waiver excluding TE) that is in Under Review Status
3. Verify action to "Upload Subsequent Documents" exists in both package list and detail view
4. Click action and verify form for each of Medicaid SPA, CHIP SPA, Initial Waiver, Renewal Waiver, Amendment Waiver; Bonus to verify each of the subtypes for waiver authorities B and B4 for each of the 3 waiver types (will affect available attachments)
5. Verify both any attachment and Additional Information is required
6. Submit and verify you are returned to the details page of the parent package. NOTE: Story did not implement viewing submitted subsequent submission details on parent package
7. Locate a TE in Under Review status and verify action is not available
